app/dash_app.py

```diff
@@ -222,9 +277,61 @@ def plot_coherence(coh_long):
     return fig
 
 
+def plot_baseline(df_baseline, df_cohfull):
+    """Plot perpendicular baseline as a function of time."""
+    bperpScatterfig = go.Scatter(x=df_baseline['second_date'],
```

Comment:
Should we not use pythonic variable names?

Reply:
Ah, the old camelCase vs snake_case debate. While technically  Pep8 allows camelCase in some cases, it does indeed recommend snake_case as best practice. As this is still a new project snake_case is indeed a better practice. Fixed